src/test/java/api/requests/scenarios/MoveRequestTests.java

```diff
@@ -386,7 +387,7 @@ public void canMoveTwoRequests()
     assertThat(interestingTimesQueue.getTotalRecords(), is(2));
   }
   
-  @Test
+  @Ignore
```

Comment:
Has this been ignored until the other pull request is merged to resolve why this is unreliable?

Reply:
Yes. You are right! As far as I know, this PR will fix this test: https://github.com/folio-org/mod-circulation/pull/310